### PR TITLE
Bac 126 add products

### PR DESCRIPTION
**Pull request by Michiel1983**
Adding new products to existing categories possible

### CODE REVIEW

File: src/main/java/nl/miwgroningen/se/ch3/bacchux/controller/ProductController.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ protected String saveOrUpdateProduct( Model model,
                 return "catalogOverview";
             }
         }
-        return "redirect:/catalog/product/" + categoryId;
+        return "redirect:/catalog/";
```
Comment (ViNguyen91) [inline src/main/java/nl/miwgroningen/se/ch3/bacchux/controller/ProductController.java:72]: After successfully added a product, it should go back to the list of the products in that category
--> return "redirect:/catalog/product/" + categoryId;

File: src/main/java/nl/miwgroningen/se/ch3/bacchux/controller/ProductController.java [hunk 2]
```diff
@@ -69,7 +69,7 @@ protected String saveOrUpdateProduct( Model model,
                 return "catalogOverview";
```
Comment (ViNguyen91) [inline src/main/java/nl/miwgroningen/se/ch3/bacchux/controller/ProductController.java:69]: Here you can check if a category is not chosen -> error message the user to choose a category

File: src/main/resources/templates/catalogOverview.html [hunk 3]
```diff
@@ -27,7 +27,8 @@ <h2>
                     <button class="list-group-item list-group-item-action categoryButton"
                             th:each="category: ${allCategories}"
```
Comment (ViNguyen91) [inline src/main/resources/templates/catalogOverview.html:28]: can sort the category here based on their names

File: src/main/resources/templates/catalogOverview.html [hunk 4]
```diff
@@ -72,8 +73,9 @@ <h5 class="modal-title" id="modalLabelCategory">New Category</h5>
                     </button>
```
Comment (ViNguyen91) [inline src/main/resources/templates/catalogOverview.html:73]: You can sort the products based on their names

**Review by ViNguyen91 — COMMENTED** [review]
1) go to /catalog -> choose add product without choosing a category -> crash
---> a message that the user has to choose a category to add a product/ no category - no product

2) the old insert database does not work
---> update the insert sql script

3) after adding a product, it should show only products in that category, not all the products in the database

4) input price: in cent or in euro?
---> ask PO

5) (Optional) categories and products are sorted

File: src/main/java/nl/miwgroningen/se/ch3/bacchux/controller/ProductController.java (hunk 1 above)
Comment (Michiel1983) [reply src/main/java/nl/miwgroningen/se/ch3/bacchux/controller/ProductController.java]: Good point, fixed

File: src/main/resources/templates/catalogOverview.html (hunk 3 above)
Comment (Michiel1983) [reply src/main/resources/templates/catalogOverview.html:29]: List is now sorted

File: src/main/resources/templates/catalogOverview.html (hunk 4 above)
Comment (Michiel1983) [reply src/main/resources/templates/catalogOverview.html:74]: List is now sorted

**Pull request merged**